x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -43,6 +45,13 @@ export class VectorStyle extends AbstractStyle {
       ...descriptor,
       ...VectorStyle.createDescriptor(descriptor.properties),
     };
+
+    this._lineColorStyleProperty = this._makeStyleProperty(vectorStyles.LINE_COLOR, this._descriptor.properties[vectorStyles.LINE_COLOR]);
```

Comment:
`_makeStyleProperty` function does not add any value. Why not just call `_makeSizeProperty` directly?  `_makeStyleProperty` is just an if/else block that is not needed since you already know the needed property at this point.